README.md

```diff
@@ -102,26 +102,34 @@ To set up **MacOS** for native development, complete the following steps:
 ### Docker
 
 For production hosting and deployment with **Docker**, use the `Dockerfile` and
-`docker-compose.yml` in the project root directory. To create a local
-development environment with **Docker**, complete the following steps:
+`docker-compose.yml` in the project root directory.
 
-- Install Docker Desktop
-- Run `docker compose -f .devcontainer/docker-compose.yml up -d`
-- Run `docker compose -f .devcontainer/docker-compose.yml exec app bin/setup`
-- Finally, run `docker compose -f .devcontainer/docker-compose.yml exec app bin/dev`
+For local development, install and launch [Docker Desktop], and run:
 
-If you are using an IDE with [support for the Development Container specification](https://containers.dev/supporting), it will run the above `docker compose` commands automatically. For **Visual Studio Code** this requires the [Dev Container extension](https://containers.dev/supporting#dev-containers).
+```shell
+docker compose -f .devcontainer/compose.yaml up -d
+docker compose -f .devcontainer/compose.yaml exec app bin/setup
+docker compose -f .devcontainer/compose.yaml exec app bin/dev
+```
+
+Within IDEs that support the [Development Containers] specification, the above
```

Comment:
Should we split DevContainers (outside codespaces) into their own section here?

Reply:
You mean like an H3 element here for just this paragaph, basically?

Reply:
Yes, because with DevContainers you do not need to launch the commands above, so I find it clearer if they have their own instructions

Reply:
I wonder if we should extract the entire "development" section to it's own document ... and/or merge the development/contributing sections into a new doc, and just link to that from readme.